### PR TITLE
Feature/discussion-chat-2

### PR DESCRIPTION
**Pull request by gabrielgomesml**
## What I did

- Create a section for users discussions with node websocket;
- Add SASS to the project and work in home responsiviness;

## How to test

- First of all make sure you are running the most updated server version in your machine. You can get it here: https://github.com/gabrielgomesml/training_project_server/pull/3
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Log into your account and you'll be redirect to the homepage;
- Click in the "Discussões" button in the navbar;
- Create a nickname to be used in the chat and after that create a new room;
- In another account (i recommend log into it with an anonymous window) do the same proccess and enter in the room that you create.

- After that, go to Home and check the FilmBox component styled with SASS and some responsiviness in the page.
- Also, check the HOC loader in Home and Admin page (make sure that you have an amount of movies and users added).

### CODE REVIEW

File: src/pages/Admin/index.tsx [hunk 1]
```diff
@@ -29,7 +31,7 @@ export const Admin: React.FC = () => {
     const data = await response.json();
```
Comment (JonasMazza) [inline src/pages/Admin/index.tsx:31]: We are still missing error handling here. Maybe you can at least add an [Error Boundary](https://reactjs.org/docs/error-boundaries.html) to catch all errors. Not the best approach, but it's better than nothing :)

File: src/pages/Home/style.ts [hunk 2]
```diff
@@ -18,6 +18,18 @@ export const ContentContainer = styled.div`
   width: 760px;
   min-height: 100vh;
   padding: 30px 30px 0px 30px;
+
+  @media (max-width: 1024px) {
```
Comment (JonasMazza) [inline src/pages/Home/style.ts:22]: Take a look at this React approach, only out of curiosity: https://github.com/artsy/fresnel

File: src/routes/index.tsx [hunk 3]
```diff
@@ -18,9 +19,9 @@ const PagesRoutes: React.FC = () => (
     <Route path="/login" component={Login} />
     <Route path="/cadastro" component={SignUp} />
     <Route path="/not-found" component={NotFound} />
-    <PrivateRoute path="/pagina-inicial" component={Home} />
+    <PrivateRoute path="/pagina-inicial" component={loader(Home)} />
```
Comment (JonasMazza) [inline src/routes/index.tsx:22]: we usually wrap the component when exporting it, like this `export default loader(Home)` for example.

**Review by JonasMazza — APPROVED** [review]
Dropped a few comments, but it is good to go.

File: src/pages/Admin/index.tsx (hunk 1 above)
Comment (gabrielgomesml) [reply src/pages/Admin/index.tsx:31]: Oh, sorry about it, i missed the error handling in some pages. I think is something that i need to be more careful about. I will update this this week in another PR!

The HOC approach i already change.

**Pull request merged**